project/src/components/boormark/boormark.tsx

```diff
@@ -1,19 +1,28 @@
+import { useState } from 'react';
+
 interface BookmarkProps {
   isSmall: boolean;
-  isActive: boolean;
+  isFavorite: boolean;
 }
 
-function Bookmark({ isSmall, isActive }: BookmarkProps): JSX.Element {
+function Bookmark({ isSmall, isFavorite }: BookmarkProps): JSX.Element {
+  let [value, setValue] = useState(isFavorite);
```

Comment:
```suggestion
  const [value, setValue] = useState(isFavorite);
```
Но вообще это надо удалить